packages/lib/getSafeRedirectUrl.ts

```diff
@@ -5,15 +5,16 @@ export const getSafeRedirectUrl = (url = "") => {
   if (!url) {
     return null;
   }
-  if (url.search(/^https?:\/\//) === -1) {
-    throw new Error("Pass an absolute URL");
-  }
-
-  const urlParsed = new URL(url);
 
-  // Avoid open redirection security vulnerability
-  if (![CONSOLE_URL, WEBAPP_URL, WEBSITE_URL].some((u) => new URL(u).origin === urlParsed.origin)) {
-    url = `${WEBAPP_URL}/`;
+  try {
+    const urlParsed = new URL(url);
+    // Avoid open redirection security vulnerability
+    if (![CONSOLE_URL, WEBAPP_URL, WEBSITE_URL].some((u) => new URL(u).origin === urlParsed.origin)) {
+      url = `${WEBAPP_URL}/`;
+    }
+  } catch (e) {
+    // If not an absolute URL
```

Comment:
Handle non absoluete URL case as well. Non absolute URLs would be safe to redirect to already.